### PR TITLE
Update perf and latest features for llm models (Aug 12)

### PR DESCRIPTION
**Pull request by skhorasganiTT**
### Ticket
N/A

### Problem description
N/A

### What's changed
- Updated perf and stable release versions for llm models on front page README
- Added time to first token numbers to front page README
- Updated MODEL_UPDATES with latest model features

### Checklist
- [x] Post commit CI passes
- [x] Blackhole Post commit (if applicable)
- [x] Model regression CI testing passes (if applicable)
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

**Comment by tt-rkim** [comment]
Have we opened P0 issues to fix these on latest main, and so we can generate a new release candidate?

**Comment by skhorasganiTT** [comment]
> Have we opened P0 issues to fix these on latest main, and so we can generate a new release candidate?

All models should be working on latest main, and if there are any that aren't, the model owners are likely looking into it. The "last verified release" refers to the last time a model owner checked that their model is functional and performant, not that commits afterwards are broken.

**Comment by skhorasganiTT** [comment]
As a follow up, if we add all of T3K demos, single-card demos, and device perf to the pre-release generation, I might be able to go and check for all models myself instead of asking individual owners for the last verified release.

**Comment by tt-rkim** [comment]
Recall single-card demos are already there. Have you not been checking? If not, have you not been checking because some others like t3000 device perf are not there?

I can add:
- t3000 demos
- single card device perf

to release and package.

**Comment by skhorasganiTT** [comment]
Thanks, it would be great if you could add those. I have been checking all of the pipelines for the models I own and asking others to do so for theirs (due to there being varying pipelines and missing tests for each), but going forwards I can check for everyone if it's all in release and package.

File: models/MODEL_UPDATES.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
-# Model Updates - July 29, 2024
```
Comment (uaydonat) [inline models/MODEL_UPDATES.md:1]: Append new features, not replace

**Review by uaydonat — APPROVED** [review]
Approved with minor edit request (MODEL_UPDATES)

File: models/MODEL_UPDATES.md (hunk 1 above)
Comment (skhorasganiTT) [reply models/MODEL_UPDATES.md:1]: Fixed

**Pull request merged**